styles.css

```diff
@@ -748,6 +824,195 @@ form {
   display: none;
 }
 
+/* Modal CSS */
+
+.modal {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  position: absolute;
+  width: 343px;
+  height: 718px;
+  left: calc(50% - 343px / 2);
+  top: 60px;
+}
+
+/* Modal Content */
+.modal-content {
+  margin: auto;
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%), 0 6px 20px 0 rgb(0 0 0 / 19%);
+  -webkit-animation-name: animatetop;
+  -webkit-animation-duration: 0.4s;
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+  background-color: #fff;
+  position: absolute;
+  width: 100%;
+  top: 15px;
+}
```

Comment:
While you did address the previous reviewer's requirement, you didn't do so properly.

<img width="1440" alt="Screen Shot 2022-07-20 at 17 04 54" src="https://user-images.githubusercontent.com/82426682/180090581-229ca77c-038b-4e34-bcc8-ad4d6c00b132.png">

<img width="812" alt="Screen Shot 2022-07-20 at 17 05 59" src="https://user-images.githubusercontent.com/82426682/180090690-977c6d0c-ca42-4c24-8629-db938b6e611a.png">

Please make sure your modal window positioning matches that of the Figma design. Make sure you fix the list of technologies' font as well.